pages/[gameId]/index.jsx

```diff
@@ -1,127 +1,23 @@
 import { useRouter } from "next/router";
-import { useState } from "react";
-import BackButton from "@/components/buttons/BackButton";
-import styled from "styled-components";
-import {
-  StyledButton,
-  StyledLinkButton,
-  StyledButtonWrapper,
-  StyledDeleteButton,
-} from "@/components/buttons/DefaultButtons";
+import GameDetailsView from "@/components/GameDetailsView";
 
-export default function GameDetails({ games, onDeleteGame, onEditGame }) {
+export default function GameDetailsWrapper({
```

Comment:
If you have searched for a good name I could recommend: 'GameDetailsPage'. This name reflects that it is a page.